unistd/file.c

```diff
@@ -290,8 +284,15 @@ int create_dev(oid_t *oid, const char *path)
 	if (!strncmp("/dev", path, 4))
 		path += 4;
 
+#if 0
```

Comment:
@agkaminski : I don't know how `create_dev()` is being used (it would be nice to add some unit tests with standard use cases.

Please comment.

Reply:
leaving as-is, to be done in separate PR